plugin/src/main/java/com/denizenscript/denizen/objects/properties/material/MaterialSwitchable.java

```diff
@@ -0,0 +1,129 @@
+package com.denizenscript.denizen.objects.properties.material;
+
+import com.denizenscript.denizen.objects.MaterialTag;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.properties.Property;
+import com.denizenscript.denizencore.objects.properties.PropertyParser;
+import org.bukkit.block.data.Powerable;
+import org.bukkit.block.data.Openable;
+import org.bukkit.block.data.type.Dispenser;
+
+public class MaterialSwitchable  implements Property {
+
+    public static boolean describes(ObjectTag material) {
+        return material instanceof MaterialTag
+                && ((MaterialTag) material).hasModernData()
+                && (((MaterialTag) material).getModernData().data instanceof Powerable
+                || ((MaterialTag) material).getModernData().data instanceof Openable
+                || ((MaterialTag) material).getModernData().data instanceof Dispenser);
+    }
+
+    public static MaterialSwitchable getFrom(ObjectTag _material) {
+        if (!describes(_material)) {
+            return null;
+        }
+        else {
+            return new MaterialSwitchable((MaterialTag) _material);
+        }
+    }
+
+    public static final String[] handledMechs = new String[] {
+            "switched"
+    };
+
+    private MaterialSwitchable(MaterialTag _material) {
+        material = _material;
+    }
+
+    MaterialTag material;
+
+    public static void registerTags() {
+
+        // <--[tag]
+        // @attribute <MaterialTag.switched>
+        // @returns ElementTag(Boolean)
+        // @group properties
+        // @description
+        // Returns if this material is switched, or not. (doors, dispensers, and other redstone materials).

```

Comment:
Meta line isn't English.
Suggestion:
`// Returns whether a switchable material (like doors, dispensers, and switches) is switched.`

Reply:
Actually:
`// Returns whether a Powerable material (like pressure plates) or an Openable material (like doors), or a dispenser is switched.`